applications/lfric_atm/fab_lfric_atm.py

```diff
@@ -156,6 +219,27 @@ def compile_fortran(self):
             '$output/science/um/atmosphere/large_scale_precipitation/*',
             [no_omp]),
             AddFlags(match="$output/science/*", flags=[real8]),
+            # jules and socrates are extracted in the science folder
```

Comment:
That's going to be a bit of work to make this really portable, i.e. don't have any compiler-specific settings in the fab build scripts :(
Best idea I have atm would be that we query the compiler to give us a specific flag (in this case whatever real8 is). But that also means that we need to add support for this to all compilers. Well, something to sweat about later :)